signinwithapplebutton/build.gradle

```diff
@@ -34,6 +34,7 @@ dependencies {
     androidTestImplementation 'androidx.test:runner:1.2.0'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
     implementation "androidx.core:core-ktx:1.0.2"
+    implementation "androidx.fragment:fragment-ktx:1.0.0"
```

Comment:
I'm not sure if I actually need this? Am I using Kotlin extensions for fragments or just using fragments in Kotlin?

Reply:
I believe it's just Kotlin extensions! We are using Fragments in Kotlin without this library, in production :)

Reply:
Yeah. I think, then, I can step down to just `androidx.fragment:fragment`. Thanks!